config/jobs/image-pushing/k8s-staging-service-apis.yaml

```diff
@@ -0,0 +1,32 @@
+postsubmits:
+  # this is the github repo we'll build from; this block needs to be repeated for each repo.
+  kubernetes-sigs/k8s-container-image-promoter:
+    - name: service-apis-postsubmit-push-to-staging
+      cluster: test-infra-trusted
```

Comment:
Jobs that run in the trusted cluster need to be defined in the trusted file

Reply:
I've followed the instructions from https://github.com/kubernetes/test-infra/blob/master/config/jobs/image-pushing/README.md

How do I add the job to the "trusted" file :sweat_smile: ?